### DISCUSSION TITLE
Grammatical error in new Discussions feature

### DISCUSSION THREAD

**Discussion by Gisgar3** [community/community#50059] [Discussions] [answered]
### Select Topic Area

Bug

### Body

There's a small grammatical error in the new feature allowing users to close Discussions. The description area for the ``Close as outdated`` option should be corrected to "no longer" instead of "not longer", just to stay grammatically correct :smiley: (error shown below)
<img width="337" alt="image" src="https://user-images.githubusercontent.com/24739483/225131304-1716e0a6-d751-4eb1-940c-35e7690b3044.png">
My OCD for grammar really kicked in 😆

**Comment by jricciardi** [comment, marked as answer]
Great catch, @Gisgar3! This bug is logged internally and we're on it 🐞 

I'm going to close this as resolved (because that's a thing we can do now!)

**Reply by jricciardi** [reply]
@Gisgar3 - thanks again for the catch. This is now fixed 😄 

<img width="464" alt="Screenshot 2023-03-15 at 8 42 42 AM" src="https://user-images.githubusercontent.com/31521880/225363438-71bd8a4e-17ed-4162-bcd5-784e92ee4781.png">
